### ISSUE TITLE
Enable ESLint for tests

### ISSUE THREAD

**Issue by josecelano** [actions/typescript-action#473]
I'm trying to enable ESLint for tests too.

But after fixing the basic errors I cannot fix this one:

```console
$ yarn eslint -c .eslintrc.json __tests__/**/*.test.ts
yarn run v1.22.17
$ /home/josecelano/Documents/github/josecelano/typescript-action/node_modules/.bin/eslint -c .eslintrc.json '__tests__/**/*.test.ts'

/home/josecelano/Documents/github/josecelano/typescript-action/__tests__/main.test.ts
  1:1  error  Filename 'main.test.ts' does not match the naming convention  filenames/match-regex

✖ 1 problem (1 error, 0 warnings)

error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

[Here](https://github.com/josecelano/typescript-action/commit/e1ddc95bf56eb93f8477e142a0e074d84bd1469a) you can see the changes I've made to the templates to support ESLint for the tests.

**Comment by DiesIrae** [comment]
Thanks @josecelano that's neat! I encountered the same problem and applied your solution

**Comment by josecelano** [comment]
I've also opened a [PR](https://github.com/actions/typescript-action/pull/474).

**Comment by ncalteen** [comment]
👋🏻 Hello! Apologies for the delay on this issue. I am going through this repository and making some large updates. This will be resolved by #732 I am going to keep this issue open though, to track and make sure that it is resolved. One of the main things to consider is where to keep tests (e.g. close to source in `src/`, or separate in `__tests__`...right now this repo opts to do the latter).

**Comment by ncalteen** [comment]
I can confirm that the updates in #732 will address this :) I've updated this issue to auto-close once that PR is merged.
